internal/page/page.go

```diff
@@ -0,0 +1,72 @@
+package page
+
+import (
+	"context"
+	"strconv"
+
+	"github.com/RHEnVision/provisioning-backend/internal/ptr"
+	"github.com/rs/zerolog"
+)
+
+type (
+	ctxKeyType  int
+	LimitOffset int
```

Comment:
Can you unexport this? Context types and keys should not be exported to prevent data manipulation from outside of the package.